File: server/worker/src/view-models/user.view-model.ts
export interface UserViewModel {
  userId: string;
  fullName: string;
  email: string;
  title?: string;
  linkedIn?: string;
  twitter?: string;
  picture?: string;
  createdAt?: Date;
  lastLogin?: Date;
  loginCount?: string;
  phase?: string;
  roles: string[];
}

File: server/core/ViewModels/UserViewModel.cs
namespace Wbs.Core.ViewModels;

public partial class UserViewModel
{
    public string UserId { get; set; }
    public string Email { get; set; }
    public string FullName { get; set; }
    public string Title { get; set; }
    public string LinkedIn { get; set; }
    public string Twitter { get; set; }
    public string Picture { get; set; }
    public DateTime? CreatedAt { get; set; }
    public DateTime? LastLogin { get; set; }
    public DateTime? UpdatedAt { get; set; }
    public int LoginCount { get; set; }
    public string Phone { get; set; }
    public string[] ShowExternally { get; set; }
}
